schemas/PerformanceTimer.json

```diff
@@ -8,15 +8,15 @@
     "name": {
       "description": "The timer that is being reported.",
       "type": "string",
-      "enum": [
-        "StartupInitialSync",
-        "StartupIncrementalSync",
-        "StartupStorePreload",
-        "StartupStoreReady",
-        "StartupLaunchScreen",
-        "InitialSyncRequest",
-        "InitialSyncParsing",
-        "NotificationsOpenEvent"
+      "oneOf": [
+        {"enum": ["StartupInitialSync"], "description": "The duration of an initial /sync request during startup (if the store has been wiped)."},
+        {"enum": ["StartupIncrementalSync"], "description": "The duration of a regular /sync request when resuming the app."},
+        {"enum": ["StartupStorePreload"], "description": "The time to preload data in the MXStore on iOS."},
+        {"enum": ["StartupStoreReady"], "description": "The time to load all data from the store (including StartupStorePreload time)."},
+        {"enum": ["StartupLaunchScreen"], "description": "How long the app launch screen is displayed for."},
+        {"enum": ["InitialSyncRequest"], "description": "The time spent waiting for a response to an initial /sync request."},
+        {"enum": ["InitialSyncParsing"], "description": "The time spent parsing the response from an initial /sync request."},
+        {"enum": ["NotificationsOpenEvent"], "description": "The time taken to display an event in the timeline that was opened from a notification."}
```

Comment:
Is it required for "enum" values to be arrays (in our use case I mean)? Can't it be a string?
EDIT I assume this is the *official* Json schema format.

Reply:
Actually you are right here. I'll update this to the
```json
{"const": "NotificationsOpenEvent", "description": "The time taken..."}
```
format, that does make more sense.

Reply:
Updated 👍